### PR TITLE
absolute URLS to relative URLs

### PR DESCRIPTION
**Pull request by slightlyoffbeat**

### CODE REVIEW

File: styles/components/_about.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
 }
 
 .about__photo {
-  background-image: url('/static/img/about/bg.png');
+  background-image: url('static/img/about/bg.png');
```
Comment (cvan) [inline styles/components/_about.scss]: don't need quotation marks, btw

File: components/About.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const About = () => (
   <div className="about" id="mission">
     <div className="about__content">
       <div>
-        <Img className="about__logo mb6" src="/static/img/logo.svg" alt="Mozilla Logo" />
+        <Img className="about__logo mb6" src="static/img/logo.svg" alt="Mozilla Logo" />
```
Comment (cvan) [inline components/About.js]: `Logo` can be `logo` (or just `Mozilla`)

Comment (cvan) [reply components/About.js]: see https://github.com/MozVR/vr-mozilla-org/pull/11/files#diff-fbd6b218b7780f00a345406be15234d1R8 too

**Comment by slightlyoffbeat** [comment]
Updated everything to do it the react way (importing the files, and then referencing. Closing.

**Pull request closed without merging**